src/apps/intel/intel10g.lua

```diff
@@ -187,6 +187,10 @@ do
    end
 end
 
+function M_sf:ingress_packet_drops ()
+   return tonumber(self.qs.QPRDC[0]())
```

Comment:
Let's leave it as a uint64.  I know it will usually be small, but usually tonumber() should only be applied to counter diffs; most places to store counter values are uint64 so they can hold the whole value.

I still don't quite understand the bit about resetting to zero.  AFAIU this register is an "RC" register (see the end of the file), and lib.hardware.register will map that to "register.readrc", which keeps its own accumulator.  Which means we don't have to mess about with counters, even; SNMP should work fine AFAIU.  If that's true then the app doesn't need to register a counter (though it can), and we can do that in some separate PR.  Sound about right to you?